app/models/post_image.rb

```diff
@@ -5,6 +5,12 @@ class PostImage < ActiveRecord::Base
 
   mount_uploader :image, PostImageUploader
 
+  def image_url
```

Comment:
True, there's a couple of ways we could improve that API, but I think they would make us change the regular carrierwave model as well, so maybe not worth it.